### PR TITLE
My story when i choose to scream

### PR DESCRIPTION
**Pull request by samtarangare**

### CODE REVIEW

**Comment by samtarangare** [comment]
Please review this, @casey-collab

File: english/scream/scream.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+
+You will do:
+
+[Jump out of joy and eat?](eat/jumpandeat.md)
+
+[Call your mom?](mom/eatwithmom.md)
+
+[Its right time to go for walk](walk/walkwiththought.md)
+
+[Keep calm and enjoy this?](http://www.gocomics.com/calvinandhobbes/)
```
Comment (casey-collab) [inline english/scream/scream.md]: It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by samtarangare** [comment]
All link works.

**Comment by samtarangare** [comment]
Please review this, @casey-collab

**Comment by casey-collab** [comment]
Looks great! :gem:

**Pull request merged**